vm_api/src/trace.rs

```diff
@@ -2,10 +2,17 @@ use fvm_ipld_encoding::ipld_block::IpldBlock;
 use fvm_shared::address::Address;
 use fvm_shared::econ::TokenAmount;
 use fvm_shared::error::{ErrorNumber, ExitCode};
+use fvm_shared::event::ActorEvent;
 use fvm_shared::{ActorID, MethodNum};
 
 type ReturnValue = Option<IpldBlock>;
 
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct EmittedEvent {
+    pub emitter: ActorID,
+    pub event: ActorEvent,
+}
```

Comment:
These changes will need to be propagated to https://github.com/anorth/fvm-workbench/blob/main/api/src/trace.rs, pending us moving this API to somewhere it can be imported by all (cc @Stebalien ). You'll probably immediately run into this need when doing gas tracing.

Reply:
Will do this as part of the FVM workbench gas analysis.

Reply:
Yeah, storing events in the trace seems reasonable.